commands/management/add_external.go

```diff
@@ -158,4 +160,5 @@ func init() {
 		EnumVar(&AddExternal.MetricsMode, metricsModes...)
 	groupHelp := fmt.Sprintf("Group name of external service (default: %s)", defaultGroupExternalExporter)
 	AddExternalC.Flag("group", groupHelp).Default(defaultGroupExternalExporter).StringVar(&AddExternal.Group)
+	AddExternalC.Flag("skip-connection-check", "Skip checks connection to exporter").BoolVar(&AddExternal.SkipConnectionCheck)
```

Comment:
Should we ask @PaulJacobs-percona what will be better to use?
Because in other places we show `Skip connection check` as help message.

Reply:
"Skip checks connection to exporter" isn't a correct sentence. I think it should be "Skip connection checks to exporter" or "Skip exporter connection checks" or "Don't check exporter connections"

Reply:
Thank you @PaulJacobs-percona